src/library/scala/collection/Iterator.scala

```diff
@@ -405,6 +413,7 @@ trait Iterator[+A] extends TraversableOnce[A] {
    *  @note    Reuse: $consumesAndProducesIterator
    */
   def filter(p: A => Boolean): Iterator[A] = new AbstractIterator[A] {
+    // TODO 2.12 - Make a full-fledged FilterImpl that will reverse sense of p
```

Comment:
I'd agree if I wasn't going to go fix these in 2.12.x right after 2.11.x is out.  As it is, it's faster and easier to grep for "TODO 2.12", and it's kind of an awkward fit for JIRA since the original improvements weren't listed there.